src/main/kotlin/no/nav/familie/baks/mottak/integrasjoner/AbstractOppgaveMapper.kt

```diff
@@ -118,6 +120,22 @@ abstract class AbstractOppgaveMapper(
             }
         }
 
+    private fun hentBehandlendeEnhetForPerson(journalpost: Journalpost): String {
+        val personIdentPåJournalpost = tilPersonIdent(journalpost.bruker!!, this.tema)
```

Comment:
Det er ingen garanti at en journalpost faktisk har bruker. En sjelden gang kommer det en journalpost uten. Nå brukes denne funksjonen kun for søknader, men funksjonen kunne ha returnert null og evt logge hvis bruker er null. I stedet vil den feile med NullPointerException som er forvirrende å finne ut av. 

Tilsvarende feil har vi i JournalhendelseKontantstøtteRutingTask. Hvor jeg til nå bare har avvikshåndtert og latt sikkerhetsnettet plukke opp oppgaven. https://familie-prosessering.intern.nav.no/service/familie-baks-mottak/task/55885550

Reply:
Veldig godt poeng @stigebil !
Jeg returnerer null isåfall, og logger.